L1Trigger/TrackFindingTracklet/interface/MatchEngineUnit.h

```diff
@@ -95,13 +101,22 @@ namespace trklet {
     //LUT for bend consistency with rinv
     const TrackletLUT& luttable_;
 
-    //Pipeline variables
-    std::pair<Tracklet*, const Stub*> tmppair_;
-    bool goodpair_;
-    bool havepair_;
+    VMStubME vmstub__, vmstub___;
```

Comment:
Usually class data members have a single underscore at the end of their names. Why do we have multiple underscores here?

Reply:
These are the pipelined variables, so each underscore corresponds to the pipeline depth.

Reply:
Please add a comment explaining that. Also please put a comment at the top of MatchEngineUnit.h explaining the purpose of this class, (since we already have MatchEngine.h).